Shared screen MP

Numenter opened the pull request:
Basic implementation for Coop and Networking. Is just a template for now, because i have no Game Vision.
I'm keen to make it something bigger.

Features:
- Drop In/Out Coop
- Shared Screen
- Local/Online Coop [WIP]
- 3D TopDown

Numenter committed:
base shared screen added

ckanitz commented on game/games/shared_screen_mp/game.cfg:
```diff
@@ -0,0 +1,7 @@
+[game]
+name ="Shared Screen MP"
```
```suggestion
name="Shared Screen MP"
```

Just formatting

Numenter committed:
shared_screen_mp/game.cfg formatting fixed

Co-authored-by: Christopher Kanitz <ckanitz@users.noreply.github.com>

Numenter committed:
left stick move and basic camera zoom added

- use left stick to move character
- zoom camera based on player distance from center
- offset on spawn to prevent collision clipping

Numenter committed:
base network added

host, join and leave
nothing is synced jet

Numenter committed:
player_pawn add/remove and move synced

Numenter committed:
some descriptions added, cleanup
